machine/machine_core/machine.py

```diff
@@ -27,8 +27,7 @@
   To quit use Ctrl+], Ctrl+5 (depending on locale)
 
 SSH ACCESS
-  $ ssh -p {ssh_port} {ssh_user}@{ssh_address}
-  Password: foobar
+  $ ssh -p {ssh_port} {ssh_user}@{ssh_address} -i bots/machine/identity
```

Comment:
Hah, I didn't even notice, as pretty well everyone in the team has that in their ~/.ssh/config. Can you please move the -i after the -p, so that the options all come before the positional arguments?